solidity/dashboard/src/services/tbtc-rewards.service.js

```diff
@@ -25,13 +26,16 @@ const fetchTBTCRewards = async (web3Context, beneficiaryAddress) => {
     transferEventSearchFilter
   )
 
+  const depositCreatedFilterParam = isEmptyArray(transferEventToBeneficiary)
```

Comment:
If my understanding is correct, I think the name of this `const` is a little misleading as we are not talking about created deposits but rather about deposits that have been successfully closed. This is out of scope of this PR though, so let's leave it as-is here.

Reply:
We scan `Created` event from `TBTCSystem` contract so `depositCreated...` it's ok I think.